### DISCUSSION TITLE
suggestion

### DISCUSSION THREAD

**Discussion by donaldli2020** [ProdigyErrorCodes/ProdigyErrorCodes#18] [Votes]
Should we change the announcements thing to the emoji mega (:mega:)?

**Comment by donaldli2020** [comment]
![](https://github.githubassets.com/images/icons/emoji/unicode/1f4e3.png)
Zoomed-in version

**Comment by afkvido** [comment]
> Should we change the announcements thing to the emoji mega (📣)?

it's a good idea, but it looks awful as hell on apple 
![image](https://user-images.githubusercontent.com/69060894/150668340-6fd7ba5e-7320-40d9-abd7-eecd79504ce7.png)

**Comment by afkvido** [comment]
any other ideas?

**Reply by donaldli2020** [reply]
idk
